tests/caasp/stack_cap.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Deploy CAP and run test app
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use parent 'caasp_controller';
+use caasp_controller;
+
+use strict;
+use testapi;
+use caasp 'script_retry';
+use utils qw(zypper_call systemctl);
+
+sub run {
+    switch_to 'xterm';
+
+    my $mip = script_output q#host -cIN master-api.openqa.test | awk '{print $NF}'#;
+    my $xip = "$mip.xip.io";
+
+    become_root;
+    systemctl 'stop apparmor';
+    systemctl 'stop SuSEfirewall2';
+
+    # Download tools
+    zypper_call 'ar -fG https://download.opensuse.org/repositories/Cloud:/Tools/SLE_12_SP3/Cloud:Tools.repo';
+    zypper_call 'ar -fG https://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12-SP3/standard/openSUSE:Backports:SLE-12-SP3.repo';
```

Comment:
Why do you need unsupported scenarios for a specific SLE version?

Reply:
I need helm & cf-cli tools - we use sles-sp3 support server.
Do you know about supported versions?